README.md

```diff
@@ -4,10 +4,30 @@ Welcome to the Chengdu chapter of [ProtoSchool](https://proto.school)!
 
 We are a group of people passionate about the decentralized web and excited to learn, teach, and share ideas. Everyone is welcome!
 
+We focus on building IPFS ecosystem (Decentralized Storage Network) cooperate with [Protocol Labs](https://protocol.ai/).  
```

Comment:
```suggestion
We focus on building the ecosystem around [IPFS](https://ipfs.io/), an open source decentralized networking protocol sponsored by [Protocol Labs](https://protocol.ai/).  
```

Reply:
I'm suggesting this change to the way IPFS is to described to avoid the  common misconception that it is identical to Filecoin, the decentralized storage network. If you are hoping to discuss both projects in your chapter, perhaps you could edit this to:
```md 
We focus on building the ecosystem around [IPFS](https://ipfs.io/) (a decentralized networking protocol), and [Filecoin](https://filecoin.io/) (a cryptocurrency-powered decentralized storage network), both open source projects sponsored by [Protocol Labs](https://protocol.ai/).
```

Reply:
Teri, I am glad to accept all your comments!